api/doc/openapi.yaml

```diff
@@ -504,6 +504,147 @@ paths:
           $ref: '#/components/responses/415'
         "500":
           $ref: '#/components/responses/500'
+  /tokens/{TokenId}:
+    get:
+      description: "Returns Token by Id"
+      operationId: getTokenById
+      parameters:
+        - name: "TokenId"
+          in: path
+          required: true
+          description: "Token Id, has format `${address}:${collection_name}:${token_name}`"
```

Comment:
creators_address

Reply:
uh, the actual PR is here https://github.com/aptos-labs/aptos-core/pull/1113 . will update that based on the comment here.